### PR TITLE
Run black on codebase

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Same thing than done in https://github.com/cds-snc/notification-admin/pull/1049

The first commit is only configuration file (easy to review) while the second one runs `black` on the codebase

### CODE REVIEW

File: scripts/run_tests.sh [hunk 1]
```diff
@@ -23,8 +23,14 @@ function display_result {
 make test-requirements
 display_result $? 1 "Requirements check"
 
+black ./app ./tests --check
```
Comment (AntoineAugusti) [inline scripts/run_tests.sh]: Changed to `black --config pyproject.toml --check .` after

File: scripts/run_tests.sh [hunk 2]
```diff
@@ -23,8 +23,14 @@ function display_result {
...
+display_result $? 2 "Code style check (Black)"
+
 flake8 .
-display_result $? 1 "Code style check"
+display_result $? 1 "Code style check (flake8)"
+
+isort --check-only ./app ./tests
```
Comment (AntoineAugusti) [inline scripts/run_tests.sh]: Changed to `isort --check-only .` after to leverage the config file

File: scripts/run_tests.sh (hunk 1 above)
Comment (smcmurtry) [reply scripts/run_tests.sh]: Nice!

File: scripts/run_tests.sh (hunk 2 above)
Comment (smcmurtry) [reply scripts/run_tests.sh]: interesting - I guess we never had an isort check on this repo?

**Review by smcmurtry — APPROVED** [review]
Looks good Antoine!

File: scripts/run_tests.sh (hunk 2 above)
Comment (AntoineAugusti) [reply scripts/run_tests.sh]: Yeah, `isort` is an addition to the API!

**Pull request merged**